### DISCUSSION TITLE
New Source: Cassandra

### DISCUSSION THREAD

**Discussion by antikilahdjs** [airbytehq/airbyte#32736] [New Connector Request]
## Tell us about the problem you're trying to solve

Hello guys, It is not a issue but I am tried check if is possible use the Cassandra as source but I am not founded nothing. I just would like to know if it is impossible due limitation or it can be expansive and not exists a possibility to use the cassandra as source. I've checked many forums and like Airbyte has a lot of destination but not source. If somebody know if is possible and exists a tool to do it I really appreciante that cuz I am new in this ETL market.

## Describe the solution you’d like
Cassandra as Source

## Describe the alternative you’ve considered or used
At this moment I dont have any alternative

## Additional context
Null

## Are you willing to submit a PR?

I dont know if have in the market or it is possible to use a cassandra as source.

**Comment by marcosmarxm** [comment]
Today Airbyte doesn't support Cassandra as a source. I changed the request to the template Airbyte uses for new sources.

**Comment by antikilahdjs** [comment]
@marcosmarxm thank you so much

**Comment by ohadfeiner** [comment]
Hey.. is this on any roadmap?

**Comment by crabio** [comment]
Hi! We are also need Cassandra Source for the Airbyte

**Comment by pgrant87** [comment]
Another request for a Cassandra source in https://github.com/airbytehq/airbyte/discussions/32812 from @komalm

[a comment by an automated account is not shown]
